modules/virt-attaching-vm-secondary-network-cli.adoc

```diff
@@ -0,0 +1,56 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/vm_networking/virt-attaching-vm-multiple-networks.adoc
+
+[id="virt-attaching-vm-secondary-network-cli_{context}"]
+= Attaching a virtual machine to a secondary network in the CLI
+
+Attach a virtual machine to a secondary network by adding a bridge to the `spec.template.spec.domain.devices.interfaces` field and specifying the network attachment definition to the `spec.template.spec.networks` field in the virtual machine configuration.
```

Comment:
- This sentence might be too long, and I don't know if it is necessary to provide this much detail in the module intro.
- s/definition to the/definition in the

Reply:
@ousleyp I've simplified the intro and moved that detail to the first step of the procedure and made that step more complete. Would you mind reading that over as it's a bit tricky. 
Also, does this make the old 'note' after the config redundant or do you think it's still useful?